### PR TITLE
Update broken links

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/api/webgl_api/webgl_best_practices/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ These WebGL 1 extensions are universally supported, and can be relied upon to be
 - WEBGL_debug_renderer_info
 - WEBGL_lose_context
 
-_(see also: <https://jdashg.github.io/misc/webgl/webgl-feature-levels.html>)_
+_(see also: <https://kdashg.github.io/misc/webgl/webgl-feature-levels.html>)_
```
Comment (MrBrain295) [inline files/en-us/web/api/webgl_api/webgl_best_practices/index.md:41]:
```suggestion
_([see also](https://kdashg.github.io/misc/webgl/webgl-feature-levels.html))_
```

Comment (OnkarRuikar) [reply files/en-us/web/api/webgl_api/webgl_best_practices/index.md:41]: I think the intention is to show the link as well (may be to inform about the location/domain).

Comment (Josh-Cena) [reply files/en-us/web/api/webgl_api/webgl_best_practices/index.md:41]: That seems less informative than the current text. We'd rather have some descriptive text of the link itself, like "See also: WebGL feature levels"

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**